lib/Service/Appointments/MailService.php

```diff
@@ -247,4 +254,97 @@ private function getSysEmail(): string {
 		$instanceName = $this->defaults->getName();
 		return \OCP\Util::getDefaultEmailAddress('appointments-noreply');
 	}
+
+	public function sendOrganizerBookingInformationEmail(Booking $booking, AppointmentConfig $config, string $calendar) {
+		/** @var IUser $user */
+		$user = $this->userManager->get($config->getUserId());
+
+		if ($user === null) {
+			throw new ServiceException('Could not find organizer');
+		}
+
+		$fromName = $user->getDisplayName();
+
+		$sys = $this->getSysEmail();
+		$message = $this->mailer->createMessage()
+			->setFrom([$sys => $fromName])
+			->setTo([$user->getEMailAddress() => $booking->getDisplayName()]);
+
+
+		$template = $this->mailer->createEMailTemplate('calendar.confirmOrganizer');
+		$template->addHeader();
+
+		//Subject
+		$subject = $this->l10n->t('You have a new appointment booking "%s" from %s', [$config->getName(), $booking->getDisplayName()]);
+		$template->setSubject($subject);
+
+		// Heading
+		$summary = $this->l10n->t('Dear %s, %s (%s) has booked an appointment with you.', [$user->getDisplayName(), $booking->getDisplayName(), $booking->getEmail()]);
```

Comment:
```suggestion
		$summary = $this->l10n->t('Dear %s, %s (%s) booked an appointment with you.', [$user->getDisplayName(), $booking->getDisplayName(), $booking->getEmail()]);
```

just a detail but again unsure about the correct grammar